src/handlers/proxyHandler.ts

```diff
@@ -206,13 +209,19 @@ export async function proxyHandler(c: Context): Promise<Response> {
       ) as Config | ShortConfig;
     }
 
+    let body;
+    if (requestContentType.startsWith(CONTENT_TYPES.GENERIC_AUDIO_PATTERN)) {
+      body = requestBinary;
+    } else if (requestContentType === CONTENT_TYPES.APPLICATION_JSON) {
+      body = store.requestFormData;
```

Comment:
Hey! This condition is incorrect. Can you please fix it?
The condition checks for application/json content type but the value being set for `body` is `store.requestFormData`.

Reply:
Updated, thanks for catching.
Moving the PR to draft to test again